.github/labeler.yml

```diff
@@ -64,33 +64,13 @@ area/gateway/lb:
   - cmd/gateway/lb/**/*
   - pkg/gateway/lb/**/*
 
-area/gateway/meta:
```

Comment:
you can also remove area/gateway & area/meta

Reply:
removed, thank you!